### DISCUSSION TITLE
p-button [outline] gives a compile error

### DISCUSSION THREAD

**Discussion by asithade** [primefaces/community#345] [PrimeNG] [answered]
When trying to use `[outline]` with p-button, either as an element or directive, I'm getting an error that it's not a known property. 

<img width="689" alt="Screenshot 2023-10-30 at 11 17 40 PM" src="https://github.com/primefaces/community/assets/3814354/f492049c-d4f1-4312-8714-122ed4095b17">

**Comment by cetincakiroglu** [comment, marked as answer]
Button doesn't have a property`outline`. It'should be `outlined`, see the demo https://primeng.org/button#outlined

**Reply by asithade** [reply]
Unfortunately, outlined doesn't work as well.
